# This message includes basic data of dynamixel

string name
uint8  id

int16  present_pwm
int16  present_current
int32  present_position
int32  present_velocity
int16  present_input_voltage
int8   present_temperature

